tests/unit/io/test_file_loader.py

```diff
@@ -19,9 +19,11 @@ def test_image_loader(image_files):
 
 @pytest.mark.skipif(sys.version_info < (3, 8), reason="requires python3.7 or higher")
 def test_file_loader(file_list):
+    filefmts = [(pd.DataFrame, "csv"), (list, "jsonl")]
```

Comment:
I dont see the need to support these old versions of python or pandas

Reply:
especially when it removes functionality

Reply:
I agree, my current plan is to get it working and checked in with the version that we have already (this bad one) and bring it up again to talk about dropping support for 3.6 for supporting excel in a different way than pandas.